Program.cs

```diff
@@ -1,17 +1,13 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License. See License.txt in the project root for license information.
 
-
 using Microsoft.Azure.Management.Fluent;
-using Microsoft.Azure.Management.ServiceBus.Fluent.Models;
 using Microsoft.Azure.Management.ResourceManager.Fluent;
 using Microsoft.Azure.Management.ResourceManager.Fluent.Core;
 using Microsoft.Azure.Management.Samples.Common;
 using Microsoft.Azure.Management.ServiceBus.Fluent;
-
 using System;
-using System.Linq;
-using System.Text;
+using System.Threading.Tasks;
 
 namespace ServiceBusWithClaimBasedAuthorization
 {

```

Comment:
I do not agree with the approach here;  we should be using an ARM template to bootstrap the Service Bus namespace and then the client library's admin client to manage child items, like the queues, topics, and subscriptions.   We shouldn't be mixing management and data plane operations into a single sample, as this is not a common application scenario.    Shifting that focus would allow us to rip out all of the extraneous references and focus clearly on a single goal for the sample.

Reply:
I think the data plane operation here is not the core operation. As it is only looks like covered in mono repro "send and receive message". Core operation should in Main() Authenticate with credentials file "my.azureauth"

Reply:
Please see comments in the project file.   I think this sample is poor quality as it was originally structured and we should either fix those issues or remove the sample.   Converting it as-is is not the right path forward, in my opinion.